f8a_worker/dispatcher/nodes.yml

```diff
@@ -277,6 +277,12 @@
       max_retry: 0
       queue: '{DEPLOYMENT_PREFIX}_{WORKER_ADMINISTRATION_REGION}_libraries_io_v0'
       storage: 'PackagePostgres'
+    - name: 'BookkeeperTask'
+      classname: 'BookkeeperTask'
+      import: 'f8a_worker.workers'
+      max_retry: 0
+      storage: 'BayesianPostgres'
```

Comment:
No need to store empty results here - see comment in `BookeperTask` implementation.